services/blockchain-indexer/config.js

```diff
@@ -129,17 +129,20 @@ config.networks = Object.freeze({
 			name: 'mainnet',
 			chainID: '00000000',
 			serviceURL: 'https://service.lisk.com',
+			snapshotUrl: 'https://snapshots.lisk.io/mainnet/service_1689588638.sql.gz',
 		},
 		{
 			name: 'testnet',
 			chainID: '01000000',
 			serviceURL: 'https://testnet-service.lisk.com',
+			snapshotUrl: 'https://snapshots.lisk.io/testnet/service_1689588642.sql.gz',
 
 		},
 		{
 			name: 'betanet',
 			chainID: '02000000',
 			serviceURL: 'https://betanet-service.lisk.com',
+			snapshotUrl: 'https://snapshots.lisk.io/betanet/service_1688983845.sql.gz',
```

Comment:
Please use the latest snapshots.
```suggestion
			snapshotURL: 'https://snapshots.lisk.com/mainnet/service.sql.gz',
		},
		{
			name: 'testnet',
			chainID: '01000000',
			serviceURL: 'https://testnet-service.lisk.com',
			snapshotURL: 'https://snapshots.lisk.com/testnet/service.sql.gz',

		},
		{
			name: 'betanet',
			chainID: '02000000',
			serviceURL: 'https://betanet-service.lisk.com',
			snapshotURL: 'https://snapshots.lisk.com/betanet/service.sql.gz',
```